### PR TITLE
Migrated the crunchy containers to Go Modules

### PR DESCRIPTION
**Pull request by ValClarkson**
Migrated the crunchy containers to Go Modules dependency management system.  Removed old dependency management system dep.

**Checklist:**

 
 - [ x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ x] Have you updated or added documentation for the change, as applicable?
 - [ x] Have you tested your changes on all related environments with successful results, as applicable?

 I didn't see any reference to dep in the documentation

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ x] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
the current management system is dep

**What is the new behavior (if this is a feature change)?**

the new management system is go modules

**Other information**:

this pr will include the removal of pcg and the removal of the old testing harness

[CH8878]

### CODE REVIEW

**Review by jkatz — COMMENTED** [review]
Looks good!

File: examples/kube/centralized-logging/efk/cleanup.sh [hunk 1]
```diff
@@ -16,13 +16,13 @@ source ${CCPROOT}/examples/common.sh
 echo_info "Cleaning up.."
 
 ${CCP_CLI?} delete --namespace=kube-system \
-    service,statefulset,deployment,pod,daemonset,configmap,secret,serviceaccount,pvc \
+    service,deployment,pod,daemonset,configmap,secret,serviceaccount,pvc \
     --selector=k8s-app=elasticsearch-logging
```
Comment (cbandy) [inline examples/kube/centralized-logging/efk/cleanup.sh]: The StatefulSets in this directory seem to be for Elasticsearch, #316. They should probably stay unless we're removing this entire example as well.

Comment (jkatz) [reply examples/kube/centralized-logging/efk/cleanup.sh]: Let's leave those in for now.

Comment (ValClarkson) [reply examples/kube/centralized-logging/efk/cleanup.sh]: sounds good, i'll put the statefulsets back in for Elasticsearch

File: examples/kube/centralized-logging/efk/run.sh [hunk 2]
```diff
@@ -18,7 +18,6 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 ${DIR}/cleanup.sh
 
-${CCP_CLI?} create -f ${DIR?}/elasticsearch-statefulset.yaml
```
Comment (cbandy) [inline examples/kube/centralized-logging/efk/run.sh]: This Elasticsearch StatefulSet needs to stay.

Comment (ValClarkson) [reply examples/kube/centralized-logging/efk/run.sh]: pushed run.sh with elasticsearch-statefulset.yaml put back.  Thanks for catching that.

File: bin/install-deps.sh [hunk 3]
```diff
@@ -46,7 +46,7 @@ tar xvzf /tmp/${FILE?} -C /tmp
 sudo cp /tmp/openshift-*/oc /usr/bin/oc
 
 # Install dep
-go get github.com/golang/dep/cmd/dep
+#go get github.com/golang/dep/cmd/dep
```
Comment (jkatz) [inline bin/install-deps.sh]: Please delete this line.

**Review by jkatz — CHANGES_REQUESTED** [review]

**Review by jkatz — APPROVED** [review]

**Comment by jkatz** [comment]
@ValClarkson It appears there are some conflicts. can you please rebase, and then I will look into merging?

**Comment by ValClarkson** [comment]
@jkatz I'm working on this now.  The above says one conflict but when i pulled in all the latest updates i have a bunch of conflicts i'm working thru.

**Pull request closed without merging**